src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java

```diff
@@ -432,6 +432,26 @@ public boolean forTextInElement(List<WebElement> elements, int index, String tex
     }
   }
 
+  public void forTextNotEmpty(WebElement element) {
+    try {
+      element.getTagName();
+    } catch (WebDriverException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)